x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/package_list_grid.tsx

```diff
@@ -201,7 +205,7 @@ function MissingIntegrationContent({
       <p>
         <FormattedMessage
           id="xpack.fleet.integrations.missing"
-          defaultMessage="Don't see an integration? Collect any logs or metrics using our {customInputsLink}. Request new integrations using our {discussForumLink}."
+          defaultMessage="Don't see an integration? Collect any logs or metrics using our {customInputsLink}, or set up {beatsTutorialLink}. Request new integrations using our {discussForumLink}."
```

Comment:
@mostlyjason @VijayDoshi Thoughts on this copy?

Reply:
How about "or add data using Beats modules"?  We haven't told the user about agent or beats yet, so they might not understand the significance of calling them "individual" Beats here.